requirements/base.in

```diff
@@ -1,6 +1,8 @@
 # Core requirements for using this package
 
+django-pyfs
 mako                                # Used by xblockutils.resources
+pypng
```

Comment:
Where are these two packages actually used?

Reply:
These were required because we are using tests suite from xblock-sdk, I've moved them to `test.txt`